pkgs/data/fonts/readexpro/default.nix

```
@@ -0,0 +1,49 @@
+{
+  pkgs ? import <nixpkgs> { },
+  lib ? import <nixpkgs/lib>,
+  fetchgit ? pkgs.fetchgit,
+  stdenvNoCC ? pkgs.stdenvNoCC,
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "readexpro";
+  version = "1.2";
+
+  src = fetchgit {
+    url = "https://github.com/ThomasJockin/readexpro.git";
+    rev = "563dfbb36ae45e52ec50829b016ce724ac2fca70";
+    sparseCheckout = [
+      "/fonts"
+    ];
+    sha256 = "sha256-WIBbw3mvSPhbwwTX+3ntCbQn4cs7ChpuTrEujR6LVTQ=";
```

Use `hash` for forwards compatibility (e.g. updating to a different hashing algorithm)
```suggestion
    hash = "sha256-WIBbw3mvSPhbwwTX+3ntCbQn4cs7ChpuTrEujR6LVTQ=";
```